src/annotator/integrations/html-side-by-side.js

```diff
@@ -1,5 +1,16 @@
 import { intersectRects, rectContains, rectIntersects } from '../util/geometry';
 
+/**
+ * CSS selectors used to find elements that are considered potentially part
+ * of the main content of a page.
+ */
+const contentSelectors = [
+  'p',
+
+  // Paragraphs in VitalSource "Great Book" format ebooks.
+  '.para',
+];
```

Comment:
I think it will be modest effort, mostly plumbing the selectors through the different layers. However it is also possible that the approach for deciding whether to consider an element changes to be based on something than selector matches.